src/main/java/org/veupathdb/service/eda/ds/plugin/pass/HistogramPlugin.java

```diff
@@ -113,7 +113,7 @@ protected void writeResults(OutputStream out, Map<String, InputStream> dataStrea
           spec.getOverlayVariable(),
           util.getVariableSpecFromList(spec.getFacetVariable(), 0),
           util.getVariableSpecFromList(spec.getFacetVariable(), 1)));
-      connection.voidEval(getVoidEvalVarMetadataMap(DEFAULT_SINGLE_STREAM_NAME, varMap));
+      connection.voidEval(getVoidEvalVariableMetadataList(varMap));
```

Comment:
This half R half Java code is fairly impenetrable.  It would help to have a comment after a line like this just saying `// variables <- ...` for example